src/Core/AutoloadInterceptor/ClassLoader.php

```diff
@@ -107,6 +107,11 @@ public function findFile($namespacedClass): false|string
 
         $filePath = Path::resolve($filePath);
 
+        foreach ($this->options->getExcludePaths() as $path) {
+            if (str_starts_with($filePath, $path)) {

```

Comment:
I think `Path::resolve($path)` would be better, so that relative paths and slight differences in paths match.